src/ddmd/argtypes.d

```diff
@@ -341,6 +341,8 @@ extern (C++) TypeTuple toArgTypes(Type t)
                     {
                         VarDeclaration f = t.sym.fields[i];
                         //printf("  [%d] %s f.type = %s\n", cast(int)i, f.toChars(), f.type.toChars());
+                        if (f.type.ty == Terror)
```

Comment:
Indeed, here is not the right place to check errors in one of the fields, should happen when semantic for the fields runs in finalizeSize or so.